src/olympia/migrations/1030-remove-obsolete-groups.sql

```diff
@@ -0,0 +1,4 @@
+SET @GROUPS_TO_REMOVE = "Developers,Developers Credits,Engagement,Monolith API,OAuth Partner: Flightdeck,Other Contributors Credits,Past Developers Credits,Persona Reviewer MOTD,QA,Revenue Stats Viewers,System Administrators";
+
+DELETE FROM groups_users WHERE group_id IN (SELECT id FROM groups WHERE FIND_IN_SET(name, @GROUPS_TO_REMOVE));
+DELETE FROM groups WHERE FIND_IN_SET(name, @GROUPS_TO_REMOVE);
```

Comment:
I'd prefer an explicit & simple list of `DELETE FROM groups WHERE name = ""';` statements than this. The list is long enough that it's hard to parse at a glance.